src/App.js

```diff
@@ -1,5 +1,5 @@
-import logo from './logo.svg';
-import './App.css';
+import logo from './assets/images/logo.svg';
```

Comment:
We could use absolute import to make the imports prettier. Implementing it at the start of the project will get rid of lots of hassles for the codebase. Details blog can be found [here](https://medium.com/hackernoon/absolute-imports-with-create-react-app-4c6cfb66c35d). 😄